src/js/components/recipient/RecipientOverview.jsx

```diff
@@ -92,6 +122,45 @@ export default class RecipientOverview extends React.Component {
             );
         }
 
+        let tooltip = null;
+        if (this.state.showInfoTooltip) {
+            tooltip = (
+                <RecipientTooltip
+                    showInfoTooltip={this.state.showInfoTooltip}
+                    closeTooltip={this.closeTooltip}
+                    content="asdf" />
```

Comment:
Is this just a placeholder?

Reply:
No, but I can remove the `content` prop, that's doing nothing atm.